magellan-legacy/src/main/java/com/wealthfront/magellan/Navigator.kt

```diff
@@ -40,17 +40,6 @@ public class Navigator internal constructor(
     }
   }
 
-  public fun addLifecycleListener(screenLifecycleListener: ScreenLifecycleListener) {
```

Comment:
I don't think we need this anymore and we can just attach it directly to the `NavigationPropogator`